gulpfile.js

```diff
@@ -189,16 +189,30 @@ gulp.task('generate-docs', function() {
   return gulpFile('api-docs.ts', docs, {src: true}).pipe(gulp.dest(PATHS.demoApiDocs));
 });
 
+gulp.task('generate-plunks', function() {
+  var getPlunker = require('./misc/plunk-gen');
+  var demoGenUtils = require('./misc/demo-gen-utils');
+
+  var componentName = 'alert';
+
+  var plunks = demoGenUtils.getDemoNames('alert').reduce(function(soFar, demoName) {
```

Comment:
This would make more sense as
```js
var plunks = demoGenUtils.getDemoNames('alert').map(demoName => {
  return {
    name: ``${componentName}/demos/${demoName}/plnkr.html`,
    source: getPlunker(componentName, demoName)
  };
});
```